src/Enums/Scope.php

```diff
@@ -26,4 +28,17 @@ public static function toIcon(self $value): string
             Scope::Everything => 'globe',
         };
     }
+
+    /**
+     * Get a string representation of a case
+     * "user", "team", or "everything"
+     */
+    public function toString(): string
```

Comment:
I feel this function isn't really necessary as the caller could call `strtolower(Enum->name)` instead of `Enum->toString()`. Looking at the code, it seems to be used 3 times, so I'd remove it in order to avoid code creep ;)